styles/globals.css

```diff
@@ -44,4 +44,36 @@ div#grid > div {
   }
 
 /* Fixes outline style for inputs on Safari */
-input:focus { outline: none; }
\ No newline at end of file
+input:focus { outline: none; }
+
+.banner {
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.banner img {
+  max-width: 40%;
+}
+
+.banner:hover img {
+  animation: wave-flag 0.5s linear infinite alternate;
+  transform-origin: top left;
+}
+
+@media (prefers-reduced-motion: reduce) {
+  .banner:hover img {
+    transition: 0.1875s cubic-bezier(0.375, 0, 0.675, 1) transform;
+    transform-origin: top left;
+  }
+}
+
+@keyframes wave-flag {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(-5deg);
+  }
+}
```

Comment:
let's use tailwind whenever possible plz

Reply:
Tailwind is powerful but it can't do animations and stuff like that

Reply:
At least that's what I think